File: Java/src/selectors/idSelectorExample.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class idSelectorExample {

	public static void main(String[] args) {
		WebDriver browser = new ChromeDriver();
		
		browser.navigate().to("https://techstepacademy.com/training-ground");
//		elements
		WebElement textInputField1 = browser.findElement(By.id("ipt1"));
		WebElement textInputField2 = browser.findElement(By.id("ipt2"));
		WebElement button1 = browser.findElement(By.id("b1"));
		WebElement button2 = browser.findElement(By.id("b2"));
		WebElement button3 = browser.findElement(By.id("b3"));
		WebElement button4 = browser.findElement(By.id("b4"));
		
//		actions
		textInputField1.sendKeys("sample text...");
		button1.click();
		browser.switchTo( ).alert( ).dismiss();
		button2.click();
		browser.switchTo( ).alert( ).dismiss();
		
		textInputField2.sendKeys("sample text...");
		button3.click();
		browser.switchTo( ).alert( ).dismiss();
		button4.click();
		browser.switchTo( ).alert( ).dismiss();
	}
}

File: C#/selectors/IdSelectorExample.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;
using System.Threading;
class IdSelectorExample
{
    static void Main()
    {
        IWebDriver browser = new ChromeDriver();
        IAlert alert;

        browser.Navigate().GoToUrl("https://techstepacademy.com/training-ground");

        // elements
        IWebElement textInputField1 = browser.FindElement(By.Id("ipt1"));
        IWebElement textInputField2 = browser.FindElement(By.Id("ipt2"));
        IWebElement button1 = browser.FindElement(By.Id("b1"));
        IWebElement button2 = browser.FindElement(By.Id("b2"));
        IWebElement button3 = browser.FindElement(By.Id("b3"));
        IWebElement button4 = browser.FindElement(By.Id("b4"));

        // actions
        textInputField1.SendKeys("sample text...");

        button1.Click();
        alert = browser.SwitchTo().Alert();
        alert.Accept();

        button2.Click();
        alert = browser.SwitchTo().Alert();
        alert.Accept();

        textInputField2.SendKeys("sample text...");

        button3.Click();
        alert = browser.SwitchTo().Alert();
        alert.Accept();

        button4.Click();
        alert = browser.SwitchTo().Alert();
        alert.Accept();

        Thread.Sleep(3000);

        browser.Quit();
    }
}